README.md

```diff
@@ -1,8 +1,94 @@
+
+Take a look at the file structure of this project, mapped out below: 
+
+```
+bin
+  |–– move
+lib
+  |–– move.rb
+spec
+  |–– move_spec.rb
+  |–– spec_helper.rb
+...
+``` 
+
+We have our `bin` directory and it contains our executable file, `move`. Remember that executable files conventionally are not given a file extension like `.rb`. 
+
+Open up `bin/move` and take a look at the following code: 
+
+```ruby
+#!/usr/bin/env ruby
+
+require_relative '../lib/move.rb'
+```
+
+First, we have our shebang line that tells the terminal which interpreter to use to execute the remainder of the file. Second, we are requiring the `move.rb` file, from within the `lib` directory. This gives our executable file access to whatever code we write in the `move.rb` file. 
+
+We'll be writing our `#display_board` and `#move` methods in `lib/move.rb` and writing the code that interacts with the command line in the `bin/move` file. 
+
+## Desired Behavior
+
+Now that we understand the broad strokes of what we've trying to accomplish, as well as where we'll be writing our code, let's take a look at an example of the functionality: 
+
+When we run our executable file with `ruby bin/move` in the terminal, inside the directory of this project, we should see something like the following: 
+
+```bash
+Welcome to Tic Tac Toe
+Where would you like to go?
+2
+   | X |   
+-----------
+   |   |   
+-----------
+   |   |   
+```
+
+Our program will: 
+
+1. Print out a welcome message.
+2. Ask the user to input the position on the board they would like to fill.
+3. Fill out that position with either an "X" or and "O".
+4. Print the updated board.
+
+Okay, we're ready to start coding!
+
+## Instructions
+
+### Part I: Defining our Methods
+
+The first part of this lab is test-driven. Run `learn` to get started and use the test output to guide you in defining the move method. 
+
+Notice that we've already given you the `#display_board` method, since we've already built that out in a previous exercise. 
+
+**Hint:** Your `#move` method must take in three arguments, the board array, the location in the board array that the player would like to fill out with an "X" or and "O", and the player's character (either "X" or "O"). If the user's input is `5`, the player wants to fill out position 5 with their character. This means that your method must fill out the correct array index with the player's character. Remember that, from the player's point of view, the board contains spaces 1-9. But, an array's indexes start their count at 0. You'll have to account for that in your `#move` method. 
+
+Once you have the tests passing, move on to part II. 
+
+### Part II: The CLI
+
+Open up `bin/move`. We're ready to code the executable portion of this program. 
+
+1. Our program should first welcome the player by outputting a friendly message to the terminal: "Welcome to Tic Tac Toe".
```

Comment:
We should make sure to remind them that the CLI has to define the starting state, ie, defining an empty board to start.